tests/foreman/cli/test_syncplan.py

```diff
@@ -102,6 +112,37 @@ def _make_sync_plan(self, options=None):
 
         return make_sync_plan(options)
 
+    def validate_repo_content(
```

Comment:
It's a bit tricky as we use different approaches for fetching the repo's info - `entities.Repository().read()` for api and `Repository.info()` for cli. Off the top of my head we can pass some optional parameter like `interface=` and act according to selected interface, or try something else. I'm ok to think a bit about that after we finally merge these tests, as right now i can't even create a new issue - neither Alex's, nor my part was even merged so there's nothing to improve yet :)